### PR TITLE
Add Support for SPM

### PR DESCRIPTION
**Pull request by janrose-lca**
Creates and adds a package description to use RxGCXMulticastDNSKit with SPM.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "RxGCXMulticastDNSKit",
+            targets: ["RxGCXMulticastDNSKit"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMinor(from: "5.0.1")),
+        .package(url: "https://github.com/grandcentrix/GCXMulticastDNSKit.git", .upToNextMinor(from: "1.3.0"))
```
Comment (PaulEhrhardt) [inline Package.swift]: I think we should link to `GCXMulticastDNSKit` beginning with a version that supports SPM. 
`1.3.0` does not support that so my proposal would be to create a `1.3.1` and then point to that as min dependency.

**Review by PaulEhrhardt — CHANGES_REQUESTED** [review]
I think it is not related to SPM but we should do a Cocoapods release as well. The same goes for the original lib `GCXMulticastDNSKit` here we also should continue supporting Cocoapods.

File: Package.swift (hunk 1 above)
Comment (janrose-lca) [reply Package.swift]: You're right. Prepared a PR for a new 1.3.1 release of GCXMulticastDNSKit

**Comment by janrose-lca** [comment]
@PaulEhrhardt 
I think I prepared everything for a new release of GCXMulticastDNSKit and bumped the version number here in preparation for a new release as well

**Review by PaulEhrhardt — APPROVED** [review]
LGTM!

**Pull request merged**